src/main/java/com/iyzipay/model/CardManagementPageInitialize.java

```diff
@@ -0,0 +1,45 @@
+package com.iyzipay.model;
+
+import com.iyzipay.HttpClient;
+import com.iyzipay.IyzipayResource;
+import com.iyzipay.Options;
+import com.iyzipay.request.CreateCardManagementPageInitializeRequest;
+
+public class CardManagementPageInitialize extends IyzipayResource {
```

Comment:
What is class name in the server? Maybe CardManagementFormInitialize could be the best.

Reply:
url has changed to pages, we will use this name for class.